tests/integration/internal/constants.go

```diff
@@ -228,6 +228,13 @@ var (
 		"otelcol_otelsvc_k8s_service_table_size",
 		"otelcol_otelsvc_k8s_owner_table_size",
 		"otelcol_exporter_send_failed_metric_points",
+		"otelcol_http_client_request_size",
+		"otelcol_http_client_response_size",
+		"otelcol_http_client_duration_bucket",
+		"otelcol_http_client_duration_sum",
+		"otelcol_http_server_response_size",
+		"otelcol_http_client_duration_count",
+		"otelcol_http_server_request_size",
```

Comment:
What are these metrics?

Reply:
It looks like otelcol exposes more metrics by default but there are also other problems, I'll take a look on this closer on Monday.